pinot-core/pom.xml

```diff
@@ -227,5 +227,25 @@
       <artifactId>lucene-analyzers-common</artifactId>
       <version>${lucene.version}</version>
     </dependency>
+    <dependency>
```

Comment:
You don't need to import them again here as they are already imported in `pinot-common`